tools/lint-all

```diff
@@ -168,6 +171,36 @@ python_rules = [
     # this rule.
     {'pattern': '% [a-zA-Z0-9_.]*\)?$',
      'description': 'Used % comprehension without a tuple'},
+    {'pattern': '\Wjson_error\([^_"\'].+\)',
```

Comment:
Hmm, can we just use `'\Wjson_error\(["\'].+\)'` as the pattern here, to check for any raw strings being passed directly into json_error?  If we do that, then we don't need the exclude list, and in fact it catches string that you missed when writing this PR.

(A similar pattern seems to work for JsonableError)

After making this tweak:
```
diff --git a/tools/lint-all b/tools/lint-all
index f84fd81..d81e375 100755
--- a/tools/lint-all
+++ b/tools/lint-all
@@ -171,34 +171,14 @@ python_rules = [
     # this rule.
     {'pattern': '% [a-zA-Z0-9_.]*\)?$',
      'description': 'Used % comprehension without a tuple'},
-    {'pattern': '\Wjson_error\([^_"\'].+\)',
+    {'pattern': '\Wjson_error\(["\'].+\)',
      'exclude': set(['tools/lint-all']),
      'exclude_line': set([
-         # function definition
-         ('zerver/lib/response.py', 'def json_error(msg, data={}, status=400):'),
-         # No need to worry about the following as the translation strings
-         # are already captured
-         ('zerver/middleware.py',
-          'return json_error(exception.to_json_error_msg(), status=status_code)'),
-         ('zerver/tornadoviews.py', 'return json_error(result["message"])'),
-         ('zerver/views/__init__.py',
-          'return json_error(data=error_data, msg=ret_error)'),
-         ('zerver/views/streams.py', 'return json_error(property_conversion)'),
-         # We can't do anything about this.
-         ('zerver/views/realm_emoji.py', 'return json_error(e.message_dict)'),
      ]),
      'description': 'json_error should only have literal strings'},
-    {'pattern': '\WJsonableError\([^_"\'].+\)',
+    {'pattern': '\WJsonableError\(["\'].+\)',
      'exclude': set(['tools/lint-all']),
      'exclude_line': set([
-         # class definition
-         ('zerver/decorator.py', 'class JsonableError(Exception):'),
-         # No need to worry about the following as the translation strings
-         # are already captured
-         ('zerver/decorator.py', 'raise JsonableError(reason % (role,))'),
-         ('zerver/decorator.py', 'raise JsonableError(error)'),
-         ('zerver/lib/actions.py', 'raise JsonableError(e.messages[0])'),
-         ('zerver/views/messages.py', 'raise JsonableError(error)'),
      ]),
      'description': 'JsonableError should only have literal strings'},
     ] + whitespace_rules

```

I get this output:
```
json_error should only have literal strings at zerver/lib/actions.py line 902:
        #     return json_error("Invalid subject name")

JsonableError should only have literal strings at zerver/tests/test_decorators.py line 28:
                raise JsonableError('13 is an unlucky number!')

```